<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>816</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>816</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	border-image: url(:/qrc/book.jpg);
	
}
QToolButton{
 border:one;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>613</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">#toolButton_3{
 border:one;
}
</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="uiqrc.qrc">
           <normaloff>:/qrc/yuan.png</normaloff>:/qrc/yuan.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#toolButton_2{
 border:one;
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="uiqrc.qrc">
           <normaloff>:/qrc/yuan.png</normaloff>:/qrc/yuan.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="baseSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#toolButton{
 border:one;
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="uiqrc.qrc">
           <normaloff>:/qrc/yuan.png</normaloff>:/qrc/yuan.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>263</x>
         <y>9</y>
         <width>990</width>
         <height>721</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>8</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#stackedWidget{
    border:one;
	background-color: rgba(255, 255, 255,200);  
    border-radius: 30px;
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QDial" name="dial">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>270</y>
           <width>50</width>
           <height>64</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>251</width>
         <height>721</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButton_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">#toolButton_9
{   
   border:one;   
	image: url(:/qrc/right.png);
}
#toolButton_9:hover 
{   
    border:one;
	background-color: rgba(255, 255, 255,200);  
    border-radius: 5px;
}
QToolButton[toolbutton=&quot;2&quot;]#toolButton_9
{
    border:one;       
	image: url(:/qrc/right.png);
}
QToolButton[toolbutton=&quot;3&quot;]#toolButton_9
{
    border:one;       
	image: url(:/qrc/left.png);
}
QToolButton[toolbutton=&quot;2&quot;]#toolButton_9:hover
{
    border:one;   
	background-color: rgba(255, 255, 255,200);  
    border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>6</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_4{
    border:one;
	background-color: rgba(255, 255, 255,200);  
    border-radius: 20px;
}</string>
          </property>
          <widget class="QToolButton" name="toolButton_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_4{
	image: url(:/qrc/individual.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">
#toolButton_5{
image: url(:/qrc/Setup.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_6{
	image: url(:/qrc/Search.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#toolButton_7{
	image: url(:/qrc/Books.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_8">
           <property name="geometry">
            <rect>
             <x>73</x>
             <y>51</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>我的</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="uiqrc.qrc"/>
 </resources>
 <connections/>
</ui>
